new_samples/README.md

```diff
@@ -0,0 +1,36 @@
+# Cadence Samples
+These are some samples to demonstrate various capabilities of Cadence client and server.  You can learn more about cadence at:
+* Cadence: https://github.com/uber/cadence
+* Cadence Client: https://github.com/uber-go/cadence-client
+
+## Prerequisite
+0. Install Cadence CLI. See instruction [here](https://cadenceworkflow.io/docs/cli/).
+1. Run Cadence Server
+
+    See instructions for running the Cadence Server: https://github.com/uber/cadence/blob/master/README.md
+2. Register a domain 
+
+    See instructions for using CLI to register a domain(name as "cadence-samples-domain"): https://cadenceworkflow.io/docs/cli/#quick-start or https://github.com/uber/cadence/blob/master/tools/cli/README.md
```

Comment:
```suggestion
1. Run the Cadence server: https://github.com/uber/cadence/blob/master/README.md
2. Register the `cadence-samples` domain: https://cadenceworkflow.io/docs/cli/#quick-start or https://github.com/uber/cadence/blob/master/tools/cli/README.md
```
maybe something like this?

or probably at least:
```
1. Run the Cadence server:  # <- two spaces at the end
   See instructions...
```
which just adds a newline rather than a full paragraph with a top margin.
(_github flavored_ markdown does not require those trailing spaces, but it's more compatible with them)

Reply:
not sure if I like the `-domain` suffix in general tbh.
- it's accompanied by `--domain ...` or `Domain: ...` everywhere so it feels redundant
- it's kinda nice that it's visibly different than the tasklist, as we need one of those too, and "cadence samples" seems reasonable there as well.  so _at least_ one of them should probably be suffixed
- but the none's suffixed and one's not

not that it matters all that much, of course.

Reply:
we can just simply use `cadence-samples` as the domain though. I'll change.